pallets/parachain-staking/src/lib.rs

```diff
@@ -585,19 +598,14 @@ pub mod pallet {
 			<MaxSelectedCandidates<T>>::put(T::MinSelectedCandidates::get());
 
 			// Choose top MaxSelectedCandidates collator candidates
-			let (_, _, collator_staked, delegator_staked) = <Pallet<T>>::select_top_candidates();
+			<Pallet<T>>::select_top_candidates();
 
 			// Start Round 0 at Block 0
 			let round: RoundInfo<T::BlockNumber> = RoundInfo::new(0u32, 0u32.into(), T::DefaultBlocksPerRound::get());
 			<Round<T>>::put(round);
 			// Snapshot total stake
 
-			Pallet::<T>::deposit_event(Event::NewRound(
-				T::BlockNumber::zero(),
-				0u32,
-				collator_staked,
-				delegator_staked,
-			));
+			Pallet::<T>::deposit_event(Event::NewRound(T::BlockNumber::zero(), 0u32));
```

Comment:
What happens if the staking pallet is added to the runtime at a later time, when the current block number is > 0? Is that even possible?

Reply:
The genesis builder function is only invoked when we build the chain spec. Thus, when the parachain-staking pallet is added to the runtime later, the initial setup needs to be taken care of in the runtime migration.

Reply:
Events arent emitted in the genesis. There is code which explicitly does not add events on block 0